src/app/core/services/auth.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
+
+import { User } from 'src/app/login/_models/user';
+
+
+@Injectable({
+    providedIn: "root"
```

Comment:
In this case I think you may not to create ServiceModule for services like we talked about the structure.